README.markdown

```diff
@@ -280,6 +285,8 @@ and works in subrequests as well.
 
 If the `-r` option is specified, then the headers will be replaced to the new values *only if* they already exist.
 
+If the `-i` options is specified, then the headers will be added to the new values *only if* they doesnt exist.
```

Comment:
The letter `i` in the `-i` option is not very descriptive. Maybe `a` is better? `a` means `add only`.